services/user.py

```diff
@@ -0,0 +1,48 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
+
+
+def create_user(username: str,
+                password: str,
+                email: str = "",
+                first_name: str = "",
+                last_name: str = "") -> User:
+    user = get_user_model().objects.create_user(username=username,
+                                                password=password,
+                                                email=email,
+                                                first_name=first_name,
+                                                last_name=last_name
+                                                )
```

Comment:
will user be created correctly with email as None etc. it seems to me it is better to make check here )